include/openenclave/attestation/attester.h

```diff
@@ -149,6 +151,48 @@ oe_result_t oe_get_attestation_certificate_with_evidence(
     uint8_t** output_cert,
     size_t* output_cert_size);
 
+/**
+ * oe_get_attestation_certificate_with_evidence_v2
+ *
+ * Similar to oe_get_attestation_certificate_with_evidence, this function
+ * generates a self-signed x.509 certificate with embedded evidence generated by
+ * an attester plugin for the enclave, but it also allows a user to pass in
+ * optional parameters.
+ *
+ * @experimental
+ *
+ * @param[in] format_id The format id of the evidence to be generated.
+ *
+ * @param[in] subject_name a string containing an X.509 distinguished
+ * name (DN) for customizing the generated certificate. This name is also used
+ * as the issuer name because this is a self-signed certificate
+ * See RFC5280 (https://tools.ietf.org/html/rfc5280) for details
```

Comment:
```suggestion
 * as the issuer name because this is a self-signed certificate.
 * See RFC5280 (https://tools.ietf.org/html/rfc5280) for details.
```